wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+- In order to reschedule, you must contact your opponents, agree on a time and be able to provide **PROOF** (by form of a screenshot/image) of the agreed reschedule to the Hosts.
+- In effort to ensure as many matches take place as possible, reschedules are allowed up to **1 HOUR BEFORE YOUR MATCH IS DUE TO BE PLAYED**. However, your opponent holds the right to **refuse a reschedule if asked less than 24 hours from the scheduled time**
+- If it is impossible to find a time to reschedule to that suits both players, both players should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
+- Matches that are uncertain to happen (for example, losers’ bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches.
+- If a match needs rescheduling to a time after a related losers’ bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed.
+
+## Links
+
+![Links](img/links.png)
+
+- [Discussion Thread](https://osu.ppy.sh/community/forums/topics/940042)
+- [UKCC Discord server](https://discord.gg/EtsU7Cw)
+- [Livestream](https://twitch.tv/osuukcc)
+- [Challonge bracket](https://challonge.com/ukcc7)
+- [Pick'em predictions website](https://pickem.hwchr.com/tournaments/15)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)**
+
+## Rules
+
+![Rules](img/rules.png)
+
+- Referees will create the lobby approximately **10 minutes** before the match is due to start.
+- Players have a 10 minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will recieve losses.
+- We take no-shows seriously, if you fail to show up to your match without valid reason, you may forfeit your right to future UKCC tournaments.
+- The referee cannot play during the match.
+
+### __**NEW**: Qualifying Rounds__
+
+- The first two rounds will be qualifying rounds, where each player will play a total of 7 maps
+- There will be 4 No Mod, 1 Hidden, 1 Hardrock and 1 Double Time map in each round
+- Your scores on each map will be added together giving you a total score
+- The top 64 players will progress from the first qualifier
+- The top 32 players will progress from the second qualifier
+- Seeding for groups will be determined from your seed based on your osu! rank (as of signup close) and your overall position in the RO64 qualifer, eg 3rd seed + 15th in qualifier = 18 points
+
+### Rolling
+
+- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](https://i.imgur.com/S0VIvni.png).
```

Comment:
```suggestion
- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
```
additionally you might want to move this image to the wiki so it's guaranteed to stay working

Reply:
How do I format this inline if i'm adding the png to a file? Is it like this and will the link work?
```suggestion
- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](img/ukcc4rollsystem.png).
```

Reply:
yep, you can test it in the github preview too